snippets/iron-elements.cson

```diff
@@ -0,0 +1,18 @@
+# Iron components snippets
+
+".text.html":
+  "iron-ajax":
+    "prefix": "iron-ajax"
+    "body": """
+    <iron-ajax
+            id="$1"
```

Comment:
I'd prefer these to be indented uniformly as two spaces. If we try to make them visually indented with their tag the indention would vary a lot. A standard I've seen from @robdodson and others in the community is to just use the single indent, so:

    <iron-ajax id="$1"
      auto
      url="$2"></iron-ajax>

I also tend to put the `id` attribute (if present) on the same line as the tag's name. That's just a personal thing so we can discuss that.